zint-typst-plugin/Cargo.toml

```diff
@@ -17,4 +17,4 @@ ciborium = "0.2.1"
 
 thiserror = "1.0"
 
-wasm-minimal-protocol = { git = "https://github.com/astrale-sharp/wasm-minimal-protocol" }
+wasm-minimal-protocol = { path = "./vendor/wasm-minimal-protocol/crates/macro" }
```

Comment:
I vendored `wasm-minimal-protocol` so we can have it in project tree, in order to build `wasm-stub` if it's not in PATH.

Reply:
This is required until [wasm-minimal-protocol#28](https://github.com/astrale-sharp/wasm-minimal-protocol/pull/28) gets merged.

Reply:
Got merged.

If [wasm-minimal-protocol#29](https://github.com/astrale-sharp/wasm-minimal-protocol/pull/29) [wasm-minimal-protocol#30](https://github.com/astrale-sharp/wasm-minimal-protocol/pull/30) get merged as well, I'll change url to upstream.

It's thankfully not something we have to update often, but I'd prefer not to have to maintain it.